en/advanced/best-practice-security.md

```diff
@@ -152,6 +153,16 @@ app.use(session({
 }))
 ```
 
+## Prevent brute-force attacks against authorisation
+
+Make sure login endpoints are protected to make private data more secure.
+
+One simple and in the same time quite powerful way is to block authorisation attempts by 2 metrics:
+1. The first is number of consecutive failed attempts by Username and IP pair. 
```

Comment:
...consecutive failed attempts by the same user name and IP address.  

Note spelling of "username".